doc/authoring_command_modules/authoring_commands.md

```diff
@@ -299,18 +335,6 @@ extra(self, dest, arg_type=None, **kwargs)
 ```
 Arguments are the same as `argument`, however this will create a new parameter whereas `argument` will not. This is useful when a reflected SDK method is missing a parameter that you need to expose in your command.
 
-***expand***
```

Comment:
Removed since we don't want anyone using this. It might even be removed. It never worked right.